install.py

```diff
@@ -8,12 +8,25 @@
 with open('photon.py') as w:
     code = w.read()
 
-code = code.replace('@PHOTON_INSTALL_PATH',f'"{os.getcwd()}/"')
+code = code.replace('@PHOTON_INSTALL_PATH', f'r"{os.getcwd()}/"')
```

Comment:
Forgot to mention, the use of a normal string here instead of a raw one was causing a SyntaxError (unicode error)